concrete/src/Page/Theme/BedrockThemeTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Concrete\Core\Page\Theme;
+
+trait BedrockThemeTrait
+{
+
+    public function registerAssets()
+    {
+        $this->requireAsset('font-awesome');
+        $this->requireAsset('jquery');
+    }
+
+    public function getThemeGridFrameworkHandle()
```

Comment:
In PHP 7.2 we can actually add return types in extended methods safely if we want to: https://3v4l.org/01O57